src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/containerized-images.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Instrument your containerized image layer
+metaDescription: A guide for instrumenting your containerized image layer with New Relic.
+freshnessValidatedDate: never
+---
+
+If you're using a containerized image for Lambda and want to monitor your application, you'll need to add New Relic to your code. After you add our pre-built layer to your image, push the image to your AWS Elastic Container Registry. From there, whenever you run your function, New Relic will run, at the same time. 
+
+Here's a guide to adding our pre-built layer to your code. 
+
+<Steps>
+<Step>
+### Add New Relic to your Dockerfile:
+New Relic offers pre-built images that include our layer's different runtimes: Java, Node.js, Python, and extension-only.
+
+1. Select the layer compatible with your specific Lambda function's runtime and region:
+ * [Java](https://gallery.ecr.aws/newrelic-lambda-layers-for-docker/newrelic-lambda-layers-java)
+ * [Node.js](https://gallery.ecr.aws/newrelic-lambda-layers-for-docker/newrelic-lambda-layers-nodejs) 
+ * [Python](https://gallery.ecr.aws/x6n7b2o2/newrelic-lambda-layers-python)
+ * [Extension-only](https://gallery.ecr.aws/newrelic-lambda-layers-for-docker/newrelic-lambda-layers-lambdaextension)
+3. **At the top of your Dockerfile** add the following code: 
+
+ <Tabs>
+        <TabsBar>
+                <TabsBarItem id="1">Dockerfile for Java</TabsBarItem>
```

Comment:
- i'd update the ids to be specific to avoid any bugs.
- since you introduce these tabs as a dockerfile already, i'd just title them with the language (e.g. <TabsBarItem id="nodejs">Nodejs</TabsBarItem>